src/index.ts

```diff
@@ -174,11 +178,39 @@ export class ChainOps {
     )
   }
 
+  /**
+   * Makes a call to watcher logging the address to the bloom
+   * @param address string of address e.g. 0x123
```

Comment:
❔ Is it useful to add a `@returns` here as well?

Reply:
done

Reply:
done